SmartDeviceLink/SDLWeatherServiceData.h

```diff
@@ -0,0 +1,79 @@
+//
+//  SDLWeatherServiceData.h
+//  SmartDeviceLink
+//
+//  Created by Nicole on 2/7/19.
+//  Copyright © 2019 smartdevicelink. All rights reserved.
+//
+
+#import "SDLRPCRequest.h"
+
+#import "SDLLocationDetails.h"
+#import "SDLWeatherData.h"
+#import "SDLWeatherAlert.h"
+
+NS_ASSUME_NONNULL_BEGIN
+
+/**
+ *  This data is related to what a weather service would provide.
+ */
+@interface SDLWeatherServiceData : SDLRPCStruct
+
+/**
```

Comment:
Should also have an init with only mandatory params

Reply:
Fixed. Added additional init.